ChangeLog

```diff
@@ -2,6 +2,11 @@ finance-yahooquote (0.26) unstable; urgency=low
 
   [ unreleased ]
 
+-- Russ Herrold <herrold@owlriver.com>  Tue, May 16 2017 12:34:54 -0400
+  * Yahoo! changed to require https connections.  For Red Hat drives, this
+  adds a new Require dependency of: perl-LWP-Protocol-https
+  * moved YahooQuote.pm to https -- noted last month in R-sig-fin ML
+
```

Comment:
This is not how (GNU) ChangeLogs have been done since the early nineties.  

Most people other than you seem to refer to computers as somehting like "system" rather than drives.    So I think I am just going to remove this paragraph and pretent you hadn't touched the file.

You do of course get credit for a PR.